tests/conftest.py

```diff
@@ -154,10 +154,10 @@ async def mobile_rw_context(event_loop):
 
 @pytest.mark.asyncio
 @pytest.yield_fixture(scope="function")
-async def mpd_private_rw_context(event_loop):
+async def vpn_private_rw_context(event_loop):
```

Comment:
This is specifically for testing private repos, but I think it may be useful for later private repo usage. We can keep it, esp since your test data file below starts with `git@github` rather than `https://github`.